src/Contrib/Newrelic/Exporter.php

```diff
@@ -54,13 +49,13 @@ public function __construct(
         SpanConverter $spanConverter = null,
         string $dataFormatVersion = Exporter::DATA_FORMAT_VERSION_DEFAULT
     ) {
+        $this->transport = (new PsrTransportFactory($client, $requestFactory, $streamFactory))->create($endpointUrl, [
+            'Api-Key' => $licenseKey,
+            'Data-Format' => 'newrelic',
+            'Data-Format-Version' => $dataFormatVersion,
+        ]);
         $this->name = $name;
-        $this->licenseKey = $licenseKey;
-        $this->dataFormatVersion = $dataFormatVersion;
-        $this->setEndpointUrl($endpointUrl);
-        $this->setClient($client);
-        $this->setRequestFactory($requestFactory);
-        $this->setStreamFactory($streamFactory);
+        $this->endpointUrl = $endpointUrl;
```

Comment:
Side note: This is only for BC / to avoid extending the scope of this PR. I don't think that the current usage of `$endpointUrl` as `host` is correct (should be the service host and not the new relic endpoint?), `host` and `service.name` should likely be fetched from the span resource instead.